packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -153,8 +155,14 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
         aria-label={t('time-picker.range-picker.backwards-time-aria-label', 'Move time range backwards')}
         variant={variant}
         onClick={onMoveBackward}
-        icon="angle-left"
+        icon="angle-double-left"
```

Comment:
hmmmm... to me, the double arrow means something different. i think of it like page controls, where a single arrow moves 1 "unit" and a double arrow moves all the units 🤔

Reply:
ah the more i look at it, the more ok i am with it. it's kinda like an old school VHS rewind/fast-forward